docs/user-guide.md

```diff
@@ -111,7 +111,8 @@ strategies:
   "PodLifeTime":
     enabled: true
     params:
-      maxPodLifeTimeSeconds: 604800 # pods run for a maximum of 7 days
+	  podLifeTime:
+        maxPodLifeTimeSeconds: 604800 # pods run for a maximum of 7 days
```

Comment:
```suggestion
      podLifeTime:
        maxPodLifeTimeSeconds: 604800 # pods run for a maximum of 7 days
```

Reply:
I guess it was missing indentation? I've updated with the latest push, hope this would've worked.